tpot/tpot.py

```diff
@@ -247,6 +246,10 @@ def fit(self, features, classes):
         None
 
         """
+        
+        if self.expert_source != None:
```

Comment:
Actually, I'm guessing this was debug code that can be removed. :-) But the point still stands when checking for `None` values.